libraries/AP_Gripper/AP_Gripper_Servo.cpp

```diff
@@ -104,8 +104,10 @@ void AP_Gripper_Servo::update_gripper()
     // Check for successful grabbed or released
     if (config.state == AP_Gripper::STATE_GRABBING && has_state_pwm(config.grab_pwm)) {
         config.state = AP_Gripper::STATE_GRABBED;
+        GCS_SEND_TEXT(MAV_SEVERITY_INFO, "Gripper load grabbed");
```

Comment:
Does this perhaps result in duplicate messages being output?  I see we're already outputting these messages higher up in the file